benchmarks/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
I think the script can be made to use POSIX sh relatively easily.

Reply:
Done.